### PR TITLE
Fix pixman compilation on OS X.

### PR DESCRIPTION
**Pull request by mvdbeek**
Need to disable mmx support.

### CODE REVIEW

File: packages/package_pixman_0_32_6/tool_dependencies.xml [hunk 1]
```diff
@@ -1,26 +1,51 @@
...
+            <actions_group>
+                <actions architecture="x86_64" os="linux">
+                    <action type="download_by_url" sha256sum="3dfed13b8060eadabf0a4945c7045b7793cc7e3e910e748a8bb0f0dc3e794904">http://cairographics.org/releases/pixman-0.32.6.tar.gz</action>
+                    <action type="set_environment_for_install">
+                        <repository name="package_libpng_1_6_7" owner="iuc" prior_installation_required="True">
+                            <package name="libpng" version="1.6.7" />
+                        </repository>
+                    </action>
+                    <action type="shell_command"><![CDATA[
+                    export LDFLAGS="-Wl,-rpath,$LIBPNG_LIB_PATH"
```
Comment (bgruening) [inline packages/package_pixman_0_32_6/tool_dependencies.xml]: Can we add an && here?

File: packages/package_pixman_0_32_6/tool_dependencies.xml [hunk 2]
```diff
@@ -1,26 +1,51 @@
...
+                <actions architecture="x86_64" os="darwin">
+                    <action type="download_by_url" sha256sum="3dfed13b8060eadabf0a4945c7045b7793cc7e3e910e748a8bb0f0dc3e794904">http://cairographics.org/releases/pixman-0.32.6.tar.gz</action>
+                    <action type="set_environment_for_install">
+                        <repository name="package_libpng_1_6_7" owner="iuc" prior_installation_required="True">
+                            <package name="libpng" version="1.6.7" />
+                        </repository>
+                    </action>
+                    <action type="shell_command"><![CDATA[
+                    export LDFLAGS="-Wl,-rpath,$LIBPNG_LIB_PATH"
+                    ./configure --prefix=$INSTALL_DIR --disable-mmx
```
Comment (bgruening) [inline packages/package_pixman_0_32_6/tool_dependencies.xml]: and here :)

**Comment by davebx** [comment]
Aha, good to know. :+1:

**Comment by bgruening** [comment]
:+1:

**Pull request merged**